features/detail.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Click a post to bring up a detail page for it
+    Implement functionality such that when I click a post body
+    or title, I am taken to a page that displays that post individually
+
+    Scenario: A visitor selecting a post from home
+        Given I am at the homepage
+        When I select a posts title or body
+        Then I am taken to a page with just the post I selected
```

Comment:
I would combine the `colorize.featue`, `detail.feature` and `markdown.feature` files into one `detail_view.feature` file.  A single feature can have multiple scenarios within it, and that allows you to use the features to represent an entire set of functionality, while scenarios exercise various aspects of that functionality.  In this case, everything related to viewing a detail page can be combined.